### PR TITLE
Workspace and CI fixes

### PR DESCRIPTION
**Pull request by henningkayser**
~~Expected to fail because of #242.~~

This resets workspace and moveit_ci branches to default ones.

### CODE REVIEW

**Comment by henningkayser** [comment]
I could move the changes related to servo into a separate PR, but then again these are all necessary to make CI succeed.

File: moveit_ros/moveit_servo/doc/servo_tutorial.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ If using the composable node, you will need to load the parameters into the "mov
 servo_params = { 'moveit_servo' : servo_yaml }
 container = ComposableNodeContainer(
             name='moveit_servo_demo_container',
-            namespace='',
+            namespace='/',
```
Comment (rhaschke) [inline moveit_ros/moveit_servo/doc/servo_tutorial.md:123]: I'm not sure about these changes. They seem to be unrelated to the other CI ones and should go into a separate PR.

**Review by rhaschke — COMMENTED** [review]
I approve the CI changes, but I cannot judge the other ones. They seem to be unrelated.

File: moveit_ros/moveit_servo/doc/servo_tutorial.md (hunk 1 above)
Comment (henningkayser) [reply moveit_ros/moveit_servo/doc/servo_tutorial.md]: These changes are required for CI to succeed, undefined namespaces are throwing an exception with the latest `rclcpp` version. I filed a separate PR here https://github.com/ros-planning/moveit2/pull/294

**Comment by henningkayser** [comment]
@rhaschke I removed the servo commits from here, the PR will probably fail CI

File: moveit_ros/moveit_servo/doc/servo_tutorial.md (hunk 1 above)
Comment (rhaschke) [reply moveit_ros/moveit_servo/doc/servo_tutorial.md]: I see. While the three commits related to .travis.yml and moveit2.repos probably should get squashed into a single commit, the other, source-related one, should be kept as a separate commit. I'm fine if you want to have that commit part of this PR.
I just couldn't judge it in the given context.

**Review by rhaschke — APPROVED** [review]

**Pull request merged**